### PR TITLE
EXERCISE_02

### PR DESCRIPTION
**Pull request by idankri**

### CODE REVIEW

File: HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class StarksGalleryActivity extends GalleryActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_starks_gallery);
+
+        String[] starks_data= {"Arya", "And such"};
+        int[] images = {}; //To be implemented
```
Comment (talboren) [inline HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java]: Why is it to be implemented? it is part of the task

File: HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import android.os.Bundle;
+
+public class StarksGalleryActivity extends GalleryActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_starks_gallery);
+
+        String[] starks_data= {"Arya", "And such"};
```
Comment (talboren) [inline HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java]: Don't mind if you had only 1 stark family member but please complete the implementation.

**Review by talboren — CHANGES_REQUESTED** [review]

File: HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java (hunk 1 above)
Comment (idankri) [reply HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java]: Seems like I mistakenly pushed an earlier debug version. I am pushing the finished version instead

File: HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java (hunk 2 above)
Comment (idankri) [reply HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java]: I re-uploaded the files (as I wrote this was an older version of my code), please check it out now :) Sorry for the inconvenience

**Review by talboren — APPROVED** [review]
Great job! loved the media player usage ;)

**Pull request merged**